src/libasr/ASR.asdl

```diff
@@ -272,6 +272,7 @@ expr
     | ComplexIm(expr arg, ttype type, expr? value)
     | DictItem(symbol a, expr key, expr? default, ttype type, expr? value)
     | CLoc(expr arg, ttype type, expr? value)
+    | GetPointer(expr arg, ttype type, expr? value)
```

Comment:
This is needed to represent, `pointer(x)` call in ASR.

Reply:
No. `Associate` associates a variable to a pointer. `GetPointer` is used for representing the operation of obtaining the pointer to a variable at lower level. Basically it represents `pointer(x)` call in ASR. Its different from associate.

Reply:
I think removal should happen in a separate PR. What do you say?

Reply:
Yes, let's keep this PR manageable. It looks like instead of Assignment, we should have `GetPointer`.